### PR TITLE
Build and remove langNames.db during install

### PR DESCRIPTION
**Pull request by TinoDidriksen**
Weirdly enough, setuptools does not allow extending the build or clean commands directly, so one has to do the work in install. This should fix issue #84.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -38,4 +54,7 @@
         'console_scripts': ['apertium-apy=apertium_apy.apy:main'],
     },
     packages=find_packages(),
+    cmdclass=dict(
```
Comment (sushain97) [inline setup.py]: The dictionary literal form (as above) is much more idiomatic. The `dict` initializer is usually used only on existing vars, e.g. a list of tuples one wishes to convert to a map.

File: setup.py [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
 from apertium_apy import apy
 
+
+class InstallHelper(install):
+    def run(self):
+        try:
+            check_call(['make', 'langNames.db'])
+        except CalledProcessError:
+            pass
+        install.run(self)
```
Comment (sushain97) [inline setup.py]: `super().run()` is preferred for its DRYness and conciseness.

Comment (TinoDidriksen) [reply setup.py]: But the instance is passed in so it's not a class hierarchy, so how can super() be relevant here? I don't actually know Python...make whatever code changes you want.

Comment (sushain97) [reply setup.py]: The bad naming convention is the real culprit here. `install` is a class. `class InstallHelper(install):` means here's a class `InstallHelper` that extends `install`. I can make the changes if you'd rather not.

**Pull request merged**